frontend/src/views/Playai.vue

```diff
@@ -32,6 +32,11 @@ export default {
     SetUpAiGame,
     GameHistory
   },
+  sockets: {
+    startGame(game) {
+      this.$router.push({ name: "Game", params: { id: game[0].id } });
+    }
+  },
```

Comment:
Do we really need it in the computer game?

Reply:
That's for ability to redirect from ai page. But I refactor that